src/configs/ensemble_config.yaml

```
@@ -0,0 +1,58 @@
+# Ensemble model configuration for house price prediction using DuckDB
+ensemble:
+  name: "house_price_ensemble_duckdb"
+  target_variable: "price"
+  method: "stacking"  # voting, stacking, blending
+  voting_method: "soft"  # soft, hard (for voting ensemble)
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)